### DISCUSSION TITLE
Can we publish the v5.0 branch to `xstate@next` so that we can test v5.0 as alpha users?

### DISCUSSION THREAD

**Discussion by huan** [statelyai/xstate#2937] [General]
I can see lots of improvements and new designs are included in the v5.0 and want to test it as soon as possible.

**Comment by davidkpiano** [comment]
Yes, soon! cc. @Andarist

**Comment by huan** [comment]
Do we have a roadmap for publishing the v5 to the `@next` NPM package now?

I'm always looking forward to the `xstate@5` to be published! :D

**Reply by Andarist** [reply]
The current plan is to publish it after landing [this PR](https://github.com/statelyai/xstate/pull/3148) (if only no other follow-up work arises from this).

**Reply by huan** [reply]
Thank you very much for letting me know this great news! :D

**Reply by huan** [reply]
How about the PR goes on?

I think it would be a great convenience if we can publish the `next` branch to the `xstate@next` NPM tag from now, and keep updating it as the PR merges!
